pyaedt/generic/spisim.py

```diff
@@ -0,0 +1,208 @@
+            self.logger.error("Failed to compute {}. Check input parameters and retry".format(parameter_name))
+            return False
+        except IndexError:
+            self.logger.error("Failed to compute {}. Check input parameters and retry".format(parameter_name))
+            return False
+
+    @pyaedt_function_handler()
+    def compute_erl(
+        self,
+        config_file=None,
+        port_order=None,
+        specify_through_ports=None,
+        bandwidth=None,
+        tdr_duration=None,
+        z_terminations=None,
+        transition_time=None,
+        fixture_delay=None,
+        input_amplitude=None,
+        ber=None,
+        pdf_bin_size=None,
+        signal_loss_factor=None,
+        permitted_reflection=None,
+        reflections_lenght=None,
+        modulation_type=None,
+    ):
+        """Compute effective return loss (erl) using Ansys SPISIM from a S-parameter file.
+
+        Parameters
+        ----------
+        config_file : str, optional
+            Configuration file to use as a reference. The default is ``None``, in
+            which case this parameter is ignored.
+        port_order : str, optional
+            Whether to use "``EvenOdd``" or "``Incremental``" numbering for ``s4p`` files.
+            Ignored if the ports are greater than 4.
+        specify_through_ports : list, optional
+            Input and Output ports on which compute the erl. Those are ordered like ``[inp, inneg, outp, outneg]``.
+            Ignored if number of ports are 4.
+        bandwidth : float, str, optional
+            Application bandwidth: inverse of one UI (unit interval). Can be a float or str with unit ("m", "g").
+            Default is ``30e9``.
+        tdr_duration : float, optional
+            TDR duration (in second): How long the TDR tailed data should be applied. Default is ``5``.
+        z_terminations : float, optional
+            Z-Terminations: termination (Z11 and Z22) when TDR is calculated. Default is ``50``.
+        transition_time : float, str, optional
+            Transition time: how fast (slew rate) input pulse transit from 0 to Vcc volt. Default is "``10p``".
+        fixture_delay : float, optional
+            Fixture delay: delay when input starts transition from 0 to Vcc. Default is ``500e-12``.
+        input_amplitude : float, optional
+            Input amplitude: Vcc volt of step input. Default is ``1.0``.
+        ber : float, optional
+            Specified BER: At what threshold ERL is calculated. Default is ``1e-4``.
+        pdf_bin_size : float, optional
+            PDF bin size: how to quantize the superimposed value. Default is ``1e-5``.
+        signal_loss_factor : float, optional
+            Signal loss factor (Beta). See SPISIM Help for info. Default is ``1.7e9``.
+        permitted_reflection : float, optional
+            Permitted reflection (Rho). See SPISIM Help for info. Default is ``0.18``.
+        reflections_lenght : float, optional
```

Comment:
Length is misspelled in the parameter name? The description needs to be something like: Length of the reflections, which are used to calculate ERL. The default is ...." Here, ERL is capitalized. We need to define on the first reference and use the abbreviation or acronym consistently thereafter.

Reply:
it is